showcase.json

```diff
@@ -3057,6 +3057,28 @@
     "youtube": "https://www.youtube.com/@usher_web3",
     "submittableId": "36108625"
   },
+  {
+  "name": "TIKITWORLD - kybra-tikit",
```

Comment:
The goal of this Ecosystem page is to showcase the product more than the core technology. Can you please update the name to "TIKITWORLD" only?